ci/cppcheck/cppcheck-new-issues.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+scriptdir="$(cd "${0%/*}" && pwd)"
+rootdir="${scriptdir%/*/*}"
+
+# shellcheck source=../../tools/function.sh
+. "${rootdir}/tools/functions.sh"
+
+
+if [ ! -f "$rootdir/cppcheck_results.txt" ] ; then
+    err "run $rootdir/tools/docker/static-analysis/cppcheck.sh first"
+fi
+
+mapfile -t issues < <(grep -E ".+:[0-9]+:[0-9]+: .+: .+ \[.+\]$"  "$rootdir/cppcheck_results.txt")
+
+status=0
+for i in "${issues[@]}"
```

Comment:
(nitpick) As happy as I am with learning the mapfile command, here it really doesn't bring anything over the traditional (and POSIX)

```
grep -E ... | \
    while read -r i; do
```